lib/matplotlib/tests/test_figure.py

```diff
@@ -1449,6 +1449,25 @@ def test_subfigure_pdf():
     fig.savefig(buffer, format='pdf')
 
 
+def test_subfigures_wspace_hspace():
+    sub_figs = plt.figure().subfigures(2, 2, hspace=0.5, wspace=0.5)
```

Comment:
Should probably test a non-square layout to ensure you didn't mix up x/y items.

Reply:
Done.